scripts/release-branch.js

```diff
@@ -18,17 +19,17 @@ const releaseBranch = 'release/' + version;
   // This asynchronous logic will happen sequentially.
   // If an error is thrown, it will break out of this
   // asynchronous function immediately and exit 1.
-
   // Create the release branch and push to Github.
   shell.exec(`git branch -D ${releaseBranch}`)
+  updateCoreVersion(version)
```

Comment:
Once updateCoreVersion is updated to be async:
```suggestion
  await updateCoreVersion(version)
```